doc/getting-started/getting-started.rst

```diff
@@ -515,10 +518,21 @@ Build ACRN
       cd ~/acrn-work/acrn-kernel
       cp kernel_config_service_vm .config
```

Comment:
Before building the acrn-kernel, we should make sure there aren't any artifacts left over from a previous build attempt by doing:
```
make distclean
```